web-ui build fails - possible issue with base nodejs-4-rhel7 image

I have been trying to run this project over the past couple of days but the web-ui keeps failing to build.  I had run this project many times 2 weeks ago and did not have any issues so I believe my steps are solid and the environment has been correctly provisioned.  Below is the error.  The only item that I see that might be an issue is the base nodejs-4-rhel7 image was updated 11 days ago so I'm guessing that may have caused this problem.  Please let me know if you need more information.

npm ERR! argv "/opt/rh/rh-nodejs4/root/usr/bin/node" "/opt/rh/rh-nodejs4/root/usr/bin/npm" "install"
npm ERR! node v4.6.2
npm ERR! npm  v2.15.1
npm ERR! code ELIFECYCLE
npm ERR! coolstore-ui@1.0.0 postinstall: `bower --config.registry.search=${BOWER_MIRROR} install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the coolstore-ui@1.0.0 postinstall script 'bower --config.registry.search=${BOWER_MIRROR} install'.
npm ERR! This is most likely a problem with the coolstore-ui package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bower --config.registry.search=${BOWER_MIRROR} install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs coolstore-ui
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls coolstore-ui
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /opt/app-root/src/npm-debug.log
error: build error: non-zero (13) exit code from registry.access.redhat.com/rhscl/nodejs-4-rhel7@sha256:8fd51091c4f919f30ed15a92b37c408ba64baf2028cc2a58dabff40f6fe86ab5

Not sure if this is the same issue:

npm WARN deprecated bower@1.8.2: ...psst! Your project can stop working at any moment because its dependencies can change. Prevent this by migrating to Yarn: https://bower.io/blog/2017/how-to-migrate-away-from-bower/
...
npm ERR! Linux 3.10.0-693.11.6.el7.x86_64
npm ERR! argv "/opt/rh/rh-nodejs4/root/usr/bin/node" "/opt/rh/rh-nodejs4/root/usr/bin/npm" "install"
npm ERR! node v4.6.2
npm ERR! npm  v2.15.1
npm ERR! code ELIFECYCLE
npm ERR! coolstore-ui@1.0.0 postinstall: `bower --config.registry.search=${BOWER_MIRROR} install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the coolstore-ui@1.0.0 postinstall script 'bower --config.registry.search=${BOWER_MIRROR} install'.
npm ERR! This is most likely a problem with the coolstore-ui package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bower --config.registry.search=${BOWER_MIRROR} install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs coolstore-ui
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls coolstore-ui
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /opt/app-root/src/npm-debug.log
error: build error: non-zero (13) exit code from registry.access.redhat.com/rhscl/nodejs-4-rhel7@sha256:b9179db6e64797cadfff1b03691c8600f6e98c28e5c2f7a8d1ef467cb6bb9d36

--- a/coolstore-ui/bower.json
+++ b/coolstore-ui/bower.json
@@ -16,6 +16,6 @@
     },
     "resolutions": {
         "angular": "1.5.10",
-        "jquery": "3.2.1"
+        "jquery": "3.3.1"
     }
 }

